src/Naneau/Obfuscator/Console/Command/ObfuscateCommand.php

```diff
@@ -171,19 +171,38 @@ public function getObfuscator()
      **/
     private function copyDir($from, $to)
     {
-        // FIXME implement native copy
-        $output = array();
-        $return = 0;
-        $command = sprintf('cp -rf %s %s', $from, $to);
-
-        exec($command, $output, $return);
-
-        if ($return !== 0)  {
+        $this->copyDirectory($from, $to);
+        
+        if (!is_dir($to))  {
             throw new \Exception('Could not copy directory');
         }
-
+        
         return $this;
     }
+    
+    /**
+     * Recursively copy a directory
+     *
+     * @param string $src
+     * @param string $dst
+     * @return void
+     **/
+    private function copyDirectory($src,$dst)
+    {
+        $dir = opendir($src);
+        @mkdir($dst);
```

Comment:
We should probably check the output of this method here, to avoid hard to debug errors down the line.